test/controllers/publishers_controller_test.rb

```
@@ -46,15 +46,75 @@ class PublishersControllerTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "re-used access link is rejected" do
+  test "re-used access link is rejected and send publisher to the expired auth token page" do
     publisher = publishers(:completed)
     url = publisher_url(publisher, token: publisher.authentication_token)
  
     get url
     assert_redirected_to home_publishers_url, "precond - publisher is logged in"
 
     get url
-    assert_redirected_to root_url, "re-used URL is rejected, publisher not logged in"
+    assert_redirected_to expired_auth_token_publishers_path(publisher_id: publisher.id), "re-used URL is rejected, publisher not logged in"
+  end
+
+  test "expired login link takes unverified publishers to dashboard" do
+    perform_enqueued_jobs do
+      post(publishers_path, params: SIGNUP_PARAMS)
+    end
+    publisher = Publisher.order(created_at: :asc).last
+
+    # expire token and attempt to cliam
+    publisher.authentication_token_expires_at = Time.now
+    publisher.save
```

minor: `save!` generally better to fail harder